pom.xml

```diff
@@ -108,6 +110,12 @@
         <dependency>
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-network-store-client</artifactId>
+            <version>${powsybl-network-store.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-network-store-iidm-impl</artifactId>
+            <version>${powsybl-network-store.version}</version>
```

Comment:
`              <!-- FIXME: to be removed at next powsybl-ws-dependencies upgrade  -->
             <dependency>
                  <groupId>com.powsybl</groupId>
                  <artifactId>powsybl-network-store-model</artifactId>
                  <version>${powsybl-network-store.version}</version>
              </dependency>` ?

Reply:
Done